docs/getting-started/reference/glossary.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Glossary
+---
+
+# Waku Docs Glossary
+
+Definitions and usage of the terminology used in the Waku ecosystem.
+
+### Bootstrapping
+
+Bootstrapping is the initial [peer discovery](#peer-discovery) process where new [nodes](#node) on the network locate and establish connections with one another.
```

Comment:
Information about bootstrap nodes must be provided out-of-band (See ... discovery section) as they are the initial entry point to the Waku network.
Once connected, other peer discovery methods can be used.